packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -2635,7 +2665,7 @@ const schema: SchemaType<DbUser> = {
   },
   
   rateLimitNextAbleToComment: {
-    type: Date,
+    type: GraphQLJSON,
```

Comment:
There might be a better type to use here, let me know if you have suggestions

Reply:
We have a function `simplSchemaToGraphQLtype` but it looks like it doesn't actually do anything currently :(